prow/plugins/trigger/trigger.go

```diff
@@ -260,3 +258,33 @@ func skipRequested(c Client, pr *github.PullRequest, skippedJobs []config.Presub
 	}
 	return errorutil.NewAggregate(errors...)
 }
+
+func getPresubmits(c Client, pr *github.PullRequest) (string, []Presubmit, error) {
+	inRepoConfigEnabled := c.Config.InRepoConfigConfiguration(org, repo).Enabled
+	// We do the status context handling here instead of in the  PluginAgent to avoid
+	// updating it once to Pending and then Success/Failed per plugin that accesses the
+	// Presubmit config
```

Comment:
Can you elaborate on this? AFAIK we never changed status contexts from the PluginAgent and loading the presubmit config should not have anything to do with posting status contexts. Additionally `trigger` never sets pending, success, or failure status contexts.

Reply:
I'm asking why we are dealing with status contexts at all. I don't know what this `prow-config-parsing` status is or why we need it. I checked the KEP and didn't see any mention of it there.